ILRepack.IntegrationTests/Scenarios/DotNet462NetStandard2/DotNet462NetStandard2.csproj

```diff
@@ -0,0 +1,83 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{90F1C943-3C23-4091-B07F-0B34748AE056}</ProjectGuid>
+    <OutputType>Exe</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>DotNet462NetStandard2</RootNamespace>
+    <AssemblyName>DotNet462NetStandard2</AssemblyName>
+    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
```

Comment:
Hmm, for .NET standard it should actually be 
```

        <TargetFramework>netstandard2.0</TargetFramework>
```
No? Otherwise it's just .NET 4.6.2

Reply:
As this PR says it's about "netstandard2 support layer for .NET 4.6.1-4.7.1"